cluster-autoscaler/clusterstate/clusterstate_test.go

```diff
@@ -781,3 +782,68 @@ func TestIsNodeStillStarting(t *testing.T) {
 func newBackoff() backoff.Backoff {
 	return backoff.NewIdBasedExponentialBackoff(InitialNodeGroupBackoffDuration, MaxNodeGroupBackoffDuration, NodeGroupBackoffResetTimeout)
 }
+
+func Test(t *testing.T) {
```

Comment:
Please name it somewhat more specific ;)

Reply:
done